src/cascadia/WindowsTerminal/IslandWindow.cpp

```diff
@@ -195,6 +195,16 @@ LRESULT IslandWindow::_OnSizing(const WPARAM wParam, const LPARAM lParam)
 
     auto clientHeight = winRect->bottom - winRect->top - nonClientSize.cy;
 
+    // If we're the quake window, prevent resizing on all sides except the
+    // bottom. This also applies to resising with the Alt+Space menu
+    if (IsQuakeWindow() && wParam != WMSZ_BOTTOM)
+    {
+        // Stuff our current window size into the lParam, and return true. This
+        // will tell User32 to use our current dimensions to resize to.
+        ::GetWindowRect(_window.get(), winRect);
+        return true;

```

Comment:
```suggestion
        return TRUE;
```
nit: probably doesn't matter, but you return `TRUE` on line 257 so figured you'd want to be consistent

Reply:
I'm actually gonna swing the other way and make them lower-case